### DISCUSSION TITLE
Default transaction isolation level

### DISCUSSION THREAD

**Discussion by Postremus** [quarkusio/quarkus#22096] [Q&A] [answered]
Hi, 

what is the default transaction isolation level, when using hibernate?
The doc just lists all possible levels, but not the default value.
https://quarkus.io/guides/datasource#quarkus-agroal_quarkus.datasource.jdbc.transaction-isolation-level

**Comment by loicmathieu** [comment, marked as answer]
It's the hibernate default which is the database default (which usually is read commited but you need to check for your database)
